framework/src/testsuite/TestUser.ts

```diff
@@ -0,0 +1,8 @@
+import { JovoUser } from '..';
+import { TestJovo } from './TestJovo';
+
+export class TestUser extends JovoUser<TestJovo> {
+  get id(): string {
+    return this.jovo?.$request?.userId || 'TestUser';
```

Comment:
Should return `undefined` instead of `TestUser` after the changes in https://github.com/jovotech/jovo-framework/pull/1031